client/src/components/client-dashboard-component/client-dashboard-filter/FilterCard.jsx

```diff
@@ -67,6 +67,14 @@ function FilterCard({
     setValues({ ...values, [prop]: event.target.value });
   };
```

Comment:
After adding a new client with an extension on their phone number and trying to filter by some phone number (I think I tried 321), the following error arises
<img width="1777" alt="Screenshot 2024-03-02 at 1 09 38 AM" src="https://github.com/uoftblueprint/tces/assets/58835213/e532226e-cbaf-4c3b-9d88-fbf1c5f75803">

Reply:
Found a semantic error. Should be good now